package_metrics/metrics.py

```diff
@@ -0,0 +1,48 @@
+import argparse
+
+from package_metrics.package_managers.pip import parse_pip
+
+
+def get_packages(package_manager):
+    parsers = {
+        "pip": parse_pip,
+    }
+    return parsers[package_manager]()
+
+
+def build_packages_table(packages):
+    def build_row(behind, name, current, latest):
+        return f"{behind:8s} {name:28s} {current:12s} {latest}"
+
+    rows = [build_row("Behind", "Package", "Latest", "Version")]
+
+    records = sorted(packages)
+    for delta, name, current, latest in records:
+        if delta:
+            behind = ".".join(str(v) for v in delta)
+        else:
+            behind = "n/a"
```

Comment:
I don't see any tests or sample output that includes this. What scenario would hit it?

nit: What do you think of changing this to `behind = ""` (empty string)?

Reply:
Yeah that is my bad. I'm looking at two different versions of the existing scripts (ones that cater to JS vs python), and this is an artifact that came out of support for JS. Given this PR is supposed to be limited to python it is probably best to remove that for now and add back in when it is actually being used/run.

Reply:
Can address the nit once we are evaluating it in a context that it is actually used 😅

Reply:
See 56b8aeb1258aa16e71dd73310e4136f9578ab461 for removal of this logic for now.